docs/modules/plugins/partials/plugin-web-app-steps.adoc

```diff
@@ -84,6 +84,41 @@ Given I am on main application page
 When I go to relative URL `/products/new`
 ----
 
+=== Web page steps
```

Comment:
- I don't see the point to have one more section, just rename the above section to have all steps interacting with pages in a single section
- it's web app plugin, so it's obvious that pages are web - in other words "web" word is redundant

Reply:
Changed